src/Microsoft.Identity.Web/WebAppExtensions/MicrosoftIdentityWebAppAuthenticationBuilder.cs

```diff
@@ -93,7 +93,8 @@ public class MicrosoftIdentityWebAppAuthenticationBuilder : MicrosoftIdentityBas
             services.AddOptions<OpenIdConnectOptions>(openIdConnectScheme)
                  .Configure<IServiceProvider>((options, serviceProvider) =>
                  {
-                     options.ResponseType = OpenIdConnectResponseType.CodeIdToken;
+                     options.ResponseType = OpenIdConnectResponseType.Code;
```

Comment:
By default when the ResponseType is Code, it seems PKCE is set to true by default.
Given MSAL.NET does not support it yet, switching it to false for the moment.